events/elections/README.md

```diff
@@ -51,28 +53,31 @@ eligibility for voting, eligibility for candidacy, maximal representation, etc.
 - Add the candidate list to the form
 - How many choices will win: This number needs to be set to the amount of open seats of a given election
 - More options, check the boxes for:
-  - Do not release results to all voters.
-  - Enable detailed ballot reporting.
-  - Allow voters to select “no opinion” for some choices.
-- Click create poll, this will send election@kubernetes.io an email with instructions.
-- It will send you a link to “Poll Control”, bookmark this generated page as this is where you will add voters and also resend ballots to people if their ballot gets lost or filtered.
-- This page is where the “Start Poll” and “Stop Poll” buttons are, start the poll.
+  - Do not release results to all voters
+  - Enable detailed ballot reporting
+  - Allow voters to select “no opinion” for some choices
+- Click create poll, this will send election@kubernetes.io an email with instructions
+- It will send you a link to “Poll Control”, bookmark this generated page as this is where you will add voters and also resend ballots to people if their ballot gets lost or filtered
+- This page is where the “Start Poll” and “Stop Poll” buttons are, start the poll
 - WARNING: This is the point of no return: 
-  - Paste in the registered voters and click add voters.
-    - It will mail the ballots to the participants.
-    - It does duplicate detection so multiple entries are fine.
-    - This might take a while and the web page will not update, this has taken up to 10m in the past as it's sending each ballot. Don't panic or refresh the page. 
-- Leave the poll open for the duration of voting.
-  - Remember to send a 24 hour reminder before closing the poll.
-  - Click "Stop poll" at the end of the election, wait a minute then refresh the page for results. 
-  - Select "Condorcet IRV" on the right hand side of the page to select the results method.
-  - Mail results of the election to the steering committee private list and await instructions. 
-- Push election results into the community repo under events/elections _after_ the Steering Commmittee has announced the results. 
+  - Paste in the registered voters and click add voters
+    - It will mail the ballots to the participants
+    - It does duplicate detection so multiple entries are fine
+    - This might take a while and the web page will not update, this has taken up to 10m in the past as it's sending each ballot. Don't panic or refresh the page
+- Leave the poll open for the duration of voting
+  - Remember to send a 24 hour reminder before closing the poll
+  - Click "Stop poll" at the end of the election, wait a minute then refresh the page for results
+  - Select "Condorcet IRV" on the right hand side of the page to select the results method
+- Reporting  
+  - Mail results of the election to the incumbent Steering Commitee members who are NOT running for election
```

Comment:
We decided to send results to continuing Steering Committee members, whose terms were not up.